source/server/getting-started/configuration/sponge-conf.rst

```diff
@@ -9,287 +9,472 @@ The global.conf file contains the global configuration settings for Sponge. This
+
+entity
+======
+
+collision-warn-size
+    Number of colliding entities in one spot before logging a warning. Set to 0 to disable
+count-warn-size
+    Number of entities in one dimension before logging a warning. Set to 0 to disable
+entity-painting-respawn-delay
+    Number of ticks before a painting is respawned on clients when their art is changed
+human-player-list-remove-delay
+    Number of ticks before the fake player entry of a human is removed from the tab list (range of 0 to 100 ticks).
+item-despawn-rate
+    Controls the time in ticks for when an item despawns.
+living-hard-despawn-range
+    The upper bounded range where living entities farther from a player will likely despawn
+living-soft-despawn-minimum-life
+    The amount of seconds before a living entity between the soft and hard despawn ranges from a player to be considered for despawning
+living-soft-despawn-range
+    The lower bounded range where living entities near a player may potentially despawn
+max-bounding-box-size
+    Maximum size of an entity's bounding box before removing it. Set to 0 to disable
+max-speed
+    Square of the maximum speed of an entity before removing it. Set to 0 to disable
+
+entity-activation-range
+=======================
+
+auto-populate
+    If 'true', newly discovered entities will be added to this config with a default value.
+defaults
+    Default activation ranges used for all entities unless overridden.
+mods
+    Per-mod overrides. Refer to the minecraft default mod for example.
+
+entity-collisions
+=================
+
+auto-populate
+    If 'true', newly discovered entities/blocks will be added to this config with a default value.
+defaults
+    Default maximum collisions used for all entities/blocks unless overridden.
+max-entities-within-aabb
+    Maximum amount of entities any given entity or block can collide with. This improves
+    performance when there are more than 8 entities on top of each other such as a 1x1
+    spawn pen. Set to 0 to disable.
+mods
+    Per-mod overrides. Refer to the minecraft default mod for example.
+defaults
+    Default maximum collisions used for all entities/blocks unless overridden.
+enabled
+    If 'false', entity collision rules for this mod will be ignored.
+defaults
+    Default maximum collisions used for all entities/blocks unless overridden.
+enabled
+    If 'false', entity collision rules for this mod will be ignored.
+
+exploits
+========
+
+prevent-creative-itemstack-name-exploit
+    Prevents an exploit in which the client sends a packet with the
+    itemstack name exceeding the string limit.
+prevent-sign-command-exploit
+    Prevents an exploit in which the client sends a packet to update a sign containing
+    commands from a player without permission.
+
+general
+=======
+
+config-dir
+    The directory for Sponge plugin configurations, relative to the
+    execution root or specified as an absolute path.
+    Note that the default: "${CANONICAL_GAME_DIR}/config"
```

Comment:
Maybe replace quotes with ` `` `

Reply:
If it's that way in the config file, we should probably leave it alone. The less editing we have to do here, the better, as the idea is that we can dump the text here in an automated fashion in future with minimum maintenance overhead.

Reply:
My intended solution was to use automation for this but either way it's not a huge change.